### DISCUSSION TITLE
parseInt problem

### DISCUSSION THREAD

**Discussion by killagu** [denysdovhan/wtfjs#191] [WTF Questions] [answered]
in [parseInt is a bad guy](https://github.com/denysdovhan/wtfjs#user-content-parseint-is-a-bad-guy) section

> parseInt('08'); // 0

but i have executed the code in chrome and node.js , the result is 8

**Comment by DimitarNestorov** [comment, marked as answer]
Since ES 2015 parseInt's second argument defaults to 10. In other words it will return 0 in an older browser.
